### PR TITLE
Grammatical fixes

### PR DESCRIPTION
**Pull request by tmsagarofficial**

### CODE REVIEW

File: LICENSE.md
Comment (hkamran80) [inline LICENSE.md:1]: The license is not created by us, so I don't think it should be edited.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: LICENSE.md
Comment (tmsagarofficial) [reply LICENSE.md:1]: Okay ill drop that commit

**Review by tmsagarofficial — COMMENTED** [review]
Check now

**Review by hkamran80 — APPROVED** [review]
LGTM!

**Comment by hkamran80** [comment]
Thank you!

**Pull request merged**